bool toBias
float64 forceMax
float64 torqueMax
---
bool success
